docs/_scripts/manifest_copy.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+echo "Create a task to copy a manifest by digest to the repo."
+export TASK_URL=$(http POST $BASE_ADDR'/docker/manifests/copy/' \
```

Comment:
i don't this will work after you merge this pr https://github.com/pulp/pulp_docker/pull/415